pkgs/scrappers/scrapper.go

```diff
@@ -59,74 +57,44 @@ func GoogleScrapper(name string) string {
 	})
 
 	var companyLink string
-	for _, link := range links {
-		answer := regexHandler.MatchCompanyLink(link, name)
+
+	for i := range links {
+		answer := regexhandler.MatchCompanyLink(links[i], name)
 		if answer == "empty" {
 			continue
 		}
 		companyLink = answer
 	}
+
 	return companyLink
 }
 
-// Scraps the company website for their email or contact-us page
-func CompanyScrapper(link string, name string) (string, string) {
-	fmt.Println("gotten link", link)
-	resp, err := http.Get(link)
-	errorHandler.HanderError("Error getting hitting company link", err)
+// Scraps the company website for their email.
+func CompanyScrapper(link, name string) (string, string) {
```

Comment:
This function name could be improved such that its purpose is clear even without the comment above it. The descriptive comments are good, keep them up, but still, `ScrapeCompanyWebsite` is more descriptive than `CompanyScrapper`